tools/azure-sdk-tools/ci_tools/versioning/version_shared.py

```diff
@@ -69,7 +69,7 @@ def get_packages(
     paths = get_setup_py_paths(args.glob_string, target_dir, additional_excludes)
 
     # Check if package is excluded if a package name param is passed
-    if package_name and not any(filter(lambda x: package_name == os.path.basename(os.path.dirname(x)), paths)):
```

Comment:
`paths` is populated by `discover_targeted_packages`, which changed to return the directory of the package, not the exact path to the `setup.py`. As a result, `os.path.dirname(x)` became unnecessary and in fact broke the targeting by name.